From fa250c4e127eab158f5f67485cb28961cbb6b294 Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Fri, 27 Jan 2017 03:07:37 -0500
Subject: [PATCH] Back out network namespace usage

To work around https://pagure.io/releng/issue/6602
---
 src/libpriv/rpmostree-bwrap.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/libpriv/rpmostree-bwrap.c b/src/libpriv/rpmostree-bwrap.c
index 40cb3dc..10dfe42 100644
--- a/src/libpriv/rpmostree-bwrap.c
+++ b/src/libpriv/rpmostree-bwrap.c
@@ -208,7 +208,6 @@ rpmostree_bwrap_new (int rootfs_fd,
                                       * but it may need some mapping work.
                                       */
                                      "--unshare-pid",
-                                     "--unshare-net",
                                      "--unshare-uts",
                                      "--unshare-ipc",
                                      "--unshare-cgroup-try",
-- 
2.9.3

